pegasus/migrations/105_add_contacts_rollup_table.rb

```diff
@@ -0,0 +1,29 @@
+Sequel.migration do
+  change do
+    create_table(:contacts_rollup, charset: 'utf8') do
+      primary_key :id, unsigned: true, null: false
+      String :email
+      Int :dashboard_user_id
```

Comment:
Good question but it can be null. IF this corresponds to a user in dashboard.users, this is their id.